photoeditor/src/main/java/ja/burhanrashid52/photoeditor/shape/ArrowPointerLocation.kt

```diff
@@ -0,0 +1,3 @@
+package ja.burhanrashid52.photoeditor.shape
+
+enum class ArrowPointerLocation { START, END, BOTH }
```

Comment:
Since this is just one Enum, Can we move this enum to `LineShape` file?

Reply:
As I understand the code correctly, `LineShape` is rather internal file for the library. We need to have access to `ArrowPointerLocation` from API so `LineShape` is not the best place for this enum:

```
ShapeBuilder.withShapeType(ShapeType.Arrow(ArrowPointerLocation.BOTH))
```

For that reason I put ArrowPointerLocation in separate file. But maybe I will put this enum in `Arrow` shape type and rename it to `PointerLocation`? This would look like this:

```
ShapeBuilder.withShapeType(ShapeType.Arrow(ShapeType.Arrow.PointerLocation.BOTH))
```